src/main/tmdb/partials/awesomeSearch/AwesomeSearchController.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * AwesomeSearchController provides controller support for inline searching
+ *
+ * @module tmdb.partials.search.AwesomeSearchController
+ *
+ * @requires angular
+ * @requires config
+ * @requires TMDBAPIService
+ *
+ * @author John Gomez <jgomez@alertlogic.com>
+ *
+ * @returns instance of the AwesomeSearchController
+ *
+ * @copyright Alert Logic, Inc 2016
+ *
+ */
+
+define( [ 'angular',
+          'config/config',
+          'tmdb/services/TMDBAPIService'],
+    function( angular, config, TMDBAPIService ) {
+        "use strict";
+
+        var AwesomeSearchController = function($scope, $rootScope, TMDBAPIService, $timeout, localStorage ) {
+            //Reference variables
+            var self = this;
+            var apiSearch = TMDBAPIService.Search();
+            var apiPerson = TMDBAPIService.Person();
+            var searchPromise;
+            self.timeWatcher = 500;
+            var limitName = 24;
+
+            var config  = angular.module("config");
+            var defaultImage = "http://www.lifeisajourney.org/wp-content/uploads/2013/03/Changed-Movie-Icon.png";
```

Comment:
I like this icon =)  I recommend you copy it into src/main/images, or some other location, and refer to it via a relative path, so that if it goes away in the future it won't effect your app.